### PR TITLE
Create config.yml for GitHub issues

### PR DESCRIPTION
**Pull request by gastaldi**
From https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

### CODE REVIEW

**Comment by gastaldi** [comment]
This is an attempt to bring back the Github issue templates when creating new issues

**Comment by dmlloyd** [comment]
The footer wanted it called `config.yml`, not `config.yaml`.  Do you think it makes a difference?

**Comment by gastaldi** [comment]
@dmlloyd nice catch, file renamed

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```
Comment (gsmet) [inline .github/ISSUE_TEMPLATE/config.yml]: Does it totally disable blank issue creation? Having it as a last resource option at the bottom was nice.

Or maybe we should have a template for it now?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+contact_links:
+  - name: Quarkus ZulipChat
```
Comment (gsmet) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
  - name: Quarkus Zulip chat
```

**Review by gsmet — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (gastaldi) [reply .github/ISSUE_TEMPLATE/config.yml]: I'm fine with enabling it.

**Comment by maxandersen** [comment]
config.yaml is not required and the issue templates are used when creating new ones....are we sure it wasn't just a temporary glitch ?

**Comment by maxandersen** [comment]
well the contact links are nice so mergning.

**Comment by gastaldi** [comment]
@maxandersen heh, looks like it was a temporary glitch, I see the templates being used again when creating a new issue. Anyway I think it's good.to merge this for the contact links at least :wink:

**Pull request merged**